pkg/sourcereader/embedded.go

```diff
@@ -103,7 +103,123 @@ func (r EmbeddedSourceReader) GetModuleInfo(modPath string, kind string) (module
 	}
 
 	reader := modulereader.Factory(kind)
-	return reader.GetInfo(modDir)
+	mi, err := reader.GetInfo(modDir)
+	mi.RequiredApis = defaultAPIList(modPath)
+	return mi, err
+}
+
+func defaultAPIList(source string) []string {
+	// API lists at
+	// https://console.cloud.google.com/apis/dashboard and
+	// https://console.cloud.google.com/apis/library
+	staticAPIMap := map[string][]string{
+		"community/modules/compute/SchedMD-slurm-on-gcp-partition": {
+			"compute.googleapis.com",
+		},
+		"community/modules/compute/htcondor-execute-point": {
+			"compute.googleapis.com",
+		},
+		"community/modules/compute/schedmd-slurm-gcp-v5-partition": {
+			"compute.googleapis.com",
+		},
+		"community/modules/database/slurm-cloudsql-federation": {
+			"bigqueryconnection.googleapis.com",
+			"sqladmin.googleapis.com",
+		},
+		"community/modules/file-system/DDN-EXAScaler": {
+			"compute.googleapis.com",
+			"deploymentmanager.googleapis.com",
+			"iam.googleapis.com",
+			"runtimeconfig.googleapis.com",
+		},
+		"community/modules/file-system/Intel-DAOS": {
```

Comment:
This is an interesting example. The source for DAOS will never be the local module. There is no wrapper. It will be something like: 

source: github.com/daos-stack/google-cloud-daos.git//terraform/modules/daos_server?ref=v0.2.1

Reply:
I understand that this particular element of the map will never be accessed, but I recorded this for completeness sake (and a starting point when we automate solution). Do you suggest removing it?

Reply:
It's a good reminder that the DAOS modules will need to updated as well to incorporate the API requirements. I support keeping it in.

Reply:
Done (no change)